traffic_ops/client/deliveryservice.go

```diff
@@ -134,7 +134,7 @@ func (to *Session) GetDeliveryServices() ([]tc.DeliveryService, ReqInf, error) {
 func (to *Session) GetDeliveryServicesByServer(id int) ([]tc.DeliveryService, ReqInf, error) {
 	var data tc.DeliveryServicesResponse
 
-	reqInf, err := get(to, fmt.Sprintf(API_SERVER_DELIVERY_SERVICES, id), &data)
+	reqInf, err := get(to, fmt.Sprintf(API_SERVER_DELIVERY_SERVICES, strconv.Itoa(id)), &data)
```

Comment:
should be all the way back to just `fmt.Sprintf(API_SERVER_DELIVERY_SERVICES, id)` instead of having the `strconv.Itoa` in there because API_SERVER_DELIVERY_SERVICES uses %d